go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da
 	github.com/google/flatbuffers v2.0.8+incompatible
 	github.com/google/go-cmp v0.5.6
+	github.com/google/uuid v1.2.0
```

Comment:
1.3.0 is available, but puts us in dependency hell.  Given this I used 1.2.0 as it was already in our build.